content/en/docs/tasks/administer-cluster/pids.md

```diff
@@ -0,0 +1,44 @@
+---
+reviewers:
+- derekwaynecarr
+- dashpole
+- RobertKrawitz
+title: Pid Limiting
```

Comment:
```suggestion
title: Limit processes available to a pod
```

Reply:
That's not what this is; the functionality of limiting processes available to a pod is pod-max-pids.  This is about limiting the number of pids available to _all_ pods collectively (specifically, reserving a certain number of pids for system and/or kubelet use).

Reply:
> If enabled, the `kubelet` argument for `pod-max-pids` will write out the configured

This does look like limiting the processes available to a pod.

@derekwaynecarr - how would you feel about having two smaller pages, specific to the tasks they're explaining, and then hyperlinking between the two?
- one page about making sure a pod doesn't use too many process IDs
- another page that describes how to ensure that there are process IDs free for kubelet / the rest of the OS?

Reply:
@RobertKrawitz reading the info as it is now i got into the same _thinking_ like @sftim .

However your explanation in the comment does makes so would you or @derekwaynecarr mind adding that info in ? 

I personally would like to see more detailed info rather than high level as not everyone knows the difference as you mentioned